_config.yml

```diff
@@ -61,3 +61,10 @@ jekyll-archives:
     month: month-archive
     day: day-archive
     tag: tag-archive
+
+webrick:
+  headers:
+    Link: <http://localhost:4000/hydra/apiDocumentation.jsonld>; rel="http://www.w3.org/ns/hydra/core#apiDocumentation"
```

Comment:
@tpluscode is there anyway to keep `localhost:4000` out of the config? Or (if it must be there) can we reuse it in the other representations?

Reply:
As for we brick's configuration, I don't think it's really relevant. This essentially has effect only when serving locally with `jekyll serve`